lib/codegen/src/regalloc/coalescing.rs

```diff
@@ -292,6 +292,9 @@ impl<'a> Context<'a> {
 
         // Insert a copy instruction at the top of `ebb`.
         let mut pos = EncCursor::new(self.func, self.isa).at_first_inst(ebb);
+        if let Some(inst) = pos.current_inst() {
+            pos.use_srcloc(inst);
+        }
```

Comment:
I think what we'd ideally like to do here is have a srcloc associated with the Ebb, so we can use that for cases like this. But we can figure that out later.